build.gradle

```diff
@@ -654,18 +657,17 @@ uiTestTargets.each { version ->
     group = 'verification'
 
     testClassesDirs = uiTest.output.classesDirs
-    classpath = sourceSets.main.output + configurations.uiTestRuntimeClasspath + uiTest.output
-    // Let's keep the XML test results in a per-version directory, this makes things easier in the CI
-    // where the UI tests are executed under a couple different IDE versions one after another.
-    // The results in the default directory (build/test-results/uiTest) would get overwritten which each subsequent test.
-    reports.junitXml.destination = file("$buildDir/test-results/uiTest/${version}")
```

Comment:
Unrel, also removed... doesn't completely fix the problem with UI test caching (since there's still https://github.com/VirtusLab/git-machete-intellij-plugin/issues/945) but gets it a bit closed.

XML test results are still in per-version directory, just sth like `build/test-results/uiTest_2021.1.1` instead of `build/test-results/uiTest/2021.1.1`... still fine, and removes a weird problem with Gradle caching (still, #945 is harder to get rid of)